### PR TITLE
#7311 choice_value callback argument can be null

### PR DESCRIPTION
**Pull request by michaellenahan**
This is my first contribution to Symfony. Thank you @deltachaos for your help today at the Hackday in Cologne.

### CODE REVIEW

**Comment by Deltachaos** [comment]
Just for the record. This fixes #7311

File: reference/forms/types/options/choice_value.rst.inc [hunk 1]
```diff
@@ -11,6 +11,9 @@ configure the value that will be sent in the API request).
 This can be a callable or a property path. See `choice_label`_ for similar usage.
 If ``null`` is used, an incrementing integer is used as the name.
 
+If you are using a callable to populate choice_value, you need to check for the
+case that the value of the field may be null.
```
Comment (xabbuh) [inline reference/forms/types/options/choice_value.rst.inc]: `null` should be enclosed with two backticks like this:

```rst
[...] field may be ``null``.
```

File: reference/forms/types/options/choice_value.rst.inc [hunk 2]
```diff
@@ -11,6 +11,9 @@ configure the value that will be sent in the API request).
 This can be a callable or a property path. See `choice_label`_ for similar usage.
 If ``null`` is used, an incrementing integer is used as the name.
 
+If you are using a callable to populate choice_value, you need to check for the
```
Comment (xabbuh) [inline reference/forms/types/options/choice_value.rst.inc]: `choice_value` should also be enclosed with double backticks

**Comment by xabbuh** [comment]
👍

**Review by javiereguiluz — APPROVED** [review]
👍  @michaellenahan thanks for your first contribution!

I've made a minor fix (we need two backticks instead of three) but your pull request was perfect otherwise. Thanks.

**Comment by xabbuh** [comment]
Thank you @michaellenahan.

**Pull request closed without merging**